src/ducks/apps/components/UninstallModal.jsx

```diff
@@ -5,8 +5,9 @@ import compose from 'lodash/flowRight'
 
 import { translate } from 'cozy-ui/transpiled/react/I18n'
 import Alerter from 'cozy-ui/transpiled/react/Alerter'
-import Button from 'cozy-ui/transpiled/react/Button'
-import Trash from 'cozy-ui/transpiled/react/Icons/Trash'
+import Buttons from 'cozy-ui/transpiled/react/Buttons'
```

Comment:
You can still use `<Button>` by importing `import Button from 'cozy-ui/transpiled/react/Buttons'`

Reply:
Is it the chosen way to use them ? Because I like to use Buttons if it Buttons I think it is more explicit.